% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
  parskip,
  oneside]{scrreprt}
\title{Schildkrötenkrebs}
\author{}
\date{\vspace{-2.5em}}

\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  pdftitle={Schildkrötenkrebs},
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage{color}
\usepackage{fancyvrb}
\newcommand{\VerbBar}{|}
\newcommand{\VERB}{\Verb[commandchars=\\\{\}]}
\DefineVerbatimEnvironment{Highlighting}{Verbatim}{commandchars=\\\{\}}
% Add ',fontsize=\small' for more characters per line
\usepackage{framed}
\definecolor{shadecolor}{RGB}{248,248,248}
\newenvironment{Shaded}{\begin{snugshade}}{\end{snugshade}}
\newcommand{\AlertTok}[1]{\textcolor[rgb]{0.94,0.16,0.16}{#1}}
\newcommand{\AnnotationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\AttributeTok}[1]{\textcolor[rgb]{0.77,0.63,0.00}{#1}}
\newcommand{\BaseNTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\BuiltInTok}[1]{#1}
\newcommand{\CharTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\CommentTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\CommentVarTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\ConstantTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ControlFlowTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\DataTypeTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{#1}}
\newcommand{\DecValTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\DocumentationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\ErrorTok}[1]{\textcolor[rgb]{0.64,0.00,0.00}{\textbf{#1}}}
\newcommand{\ExtensionTok}[1]{#1}
\newcommand{\FloatTok}[1]{\textcolor[rgb]{0.00,0.00,0.81}{#1}}
\newcommand{\FunctionTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\ImportTok}[1]{#1}
\newcommand{\InformationTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\newcommand{\KeywordTok}[1]{\textcolor[rgb]{0.13,0.29,0.53}{\textbf{#1}}}
\newcommand{\NormalTok}[1]{#1}
\newcommand{\OperatorTok}[1]{\textcolor[rgb]{0.81,0.36,0.00}{\textbf{#1}}}
\newcommand{\OtherTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{#1}}
\newcommand{\PreprocessorTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textit{#1}}}
\newcommand{\RegionMarkerTok}[1]{#1}
\newcommand{\SpecialCharTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\SpecialStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\StringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\VariableTok}[1]{\textcolor[rgb]{0.00,0.00,0.00}{#1}}
\newcommand{\VerbatimStringTok}[1]{\textcolor[rgb]{0.31,0.60,0.02}{#1}}
\newcommand{\WarningTok}[1]{\textcolor[rgb]{0.56,0.35,0.01}{\textbf{\textit{#1}}}}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{5}
\usepackage[miktex]
\usepackage[greek, ngerman, main=english]{babel}
\usepackage[utf8]{inputenc}
\usepackage[T1]{fontenc}
\usepackage{lmodern}
\usepackage[onehalfspacing]{setspace}
\usepackage[left=2.50cm, right=2.50cm, top=2.50cm, bottom=2.50cm, bindingoffset=10mm, includehead, includefoot]{geometry}
\usepackage[headsepline]{scrlayer-scrpage}
\usepackage{url}
\usepackage[backend=biber, style=authoryear, giveninits=true, maxbibnames=99, uniquename=init, maxcitenames=2, hyperref=true, date=year]{biblatex}
\usepackage{xpatch}
\usepackage{csquotes}
\usepackage{amsmath}
\usepackage{listings}
\usepackage{booktabs}
\usepackage{longtable}
\usepackage{multirow}
\usepackage{rotating}
\usepackage{subfigure}
\usepackage{graphicx}
\usepackage{float}
\usepackage{acronym}
\usepackage{lipsum}
\usepackage{scrhack}
\usepackage{amsmath}
\emergencystretch=50pt
\clubpenalty = 10000
\widowpenalty = 10000
\displaywidowpenalty = 10000
\automark[section]{chapter}
\renewcommand*{\chaptermarkformat}{}
\renewcommand*{\sectionmarkformat}{}
\setkomafont{title}{\sffamily}
\setkomafont{disposition}{\usekomafont{title}}
\setkomafont{author}{\usekomafont{title}}
\setkomafont{date}{\usekomafont{title}}
\setkomafont{caption}{\sffamily\small}
\setkomafont{captionlabel}{\usekomafont{caption}\bfseries\small}
\setkomafont{pagehead}{\normalfont\scshape}
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi

\begin{document}
\maketitle

\begin{titlepage}
\centering
    {\Large Ruprecht-Karls-Universität Heidelberg\\
        Fakultät für Biowissenschaften\\
        Bachelorstudiengang Molekulare Biotechnologie\\}

    {\vspace{\stretch{2}}}
    {\usekomafont{title}

        {\Huge ksdflsdjf}

        {\Huge sfdsfd}

        {\Huge sfsf}

    }

    \vspace{\stretch{2}}
    {\Large Data Science Project SoSe 2022}

    \vspace{\stretch{2}}

    {\Large
        \begin{tabular}{rl}
            Autoren & Anna Lange, David Matuschek, Jakob Then, Maren Schneider\\
            Geburtsort & Heidelberg\\
            Abgabetermin &20.07.2022\\
        \end{tabular}
    }

    \vspace{\stretch{1}}

\end{titlepage}

\tableofcontents

`\texttt{\#}\{r child = ``chapter/abstract.Rmd''\}

\begin{verbatim}

# Introduction

# Introduction

2019 starben 230,242 Menschen in Deutschland an Krebs \[Quelle\] \[[Krebsrate und Krebs-Sterberate in Deutschland (krebsinformationsdienst.de)](https://www.krebsinformationsdienst.de/tumorarten/grundlagen/krebsstatistiken.php)\]. Um Tumore zu erkennen und eine besserer Behandlung zu finden, ist die Entwicklung neuer Behandlungsmethoden von hoher Wichtigkeit. Dazu ist es essenziell die mutationstechnischen uRsachen für Krebsentwicklung zu identifizieren. Dafür können transcriptomic profiling methods wie RNA-seq verendet werden.

The provided data in the following analysis originates from a transcriptomic profiling methods like RNA-seq. Transcriptomic profiling sequences all the RNA that has been generated by transcription of the cells DNA. The difference to sequencing of DNA is, that it only sequences those genes, that are going to be expressed in that cell.

#### RNA-seq

RNA-seq is performed by cleaning of RNA, fragmentation, translation of RNA to cDNA, sequencing of cNDA and comparing with the reference genome. The advantage of RNA-seq is that it includes information about gene expression, that is especially important in the analysis of tumors such as epigenetic changes (e.g. epigenetic gene silencing) or fusion proteins.

The results from RNA-seq used for the analysis originate from the cancer genome atlas (TCGA)

## Thyroid carcinoma 

Thyroid carcinoma (THCA) incidence increased dramatically over the past few years (<https://jamanetwork.com/data/journals/intemed/936342/m_ied170005f1.png>) REFERENCE, deswegen schauen wir uns die Gene, die THCA verursachen im unserer Analyse näher an. Allerdings sind nur 1% der deutschen Tumoren THCA.

Thyroid cancers werden in verschiedene Typen aufgeteilt, der häufigste Typ ist papillary thyroid cancer (PTC) with 80% of total Thyroid cancers. The most common mutation in PTC is the V600E-Mutation of the RAF-Kinase, what causes a constant activation and by intracellular signaling it promotes tumor cell growth and thereby growth of the tumor.

Die Aufgabe der Schilddrüse ist es, Hormone zu sythetisieren und somit Köpertemperatur und Metabolismus zu kontrollieren. Thyroxine spielt dabei eine wichtige Rolle im Metabolismus, indem es die metabolic rate stimuliert. [Thyroxine - Higher - Coordination and control - The human endocrine system - Edexcel - GCSE Biology (Single Science) Revision - Edexcel - BBC Bitesize](https://www.bbc.co.uk/bitesize/guides/z3gxb82/revision/3#:~:text=Thyroxine%20is%20produced%20from%20the%20thyroid%20gland%2C%20which,development.%20Its%20levels%20are%20controlled%20by%20negative%20feedback.) xxx. Was auch als eine underactive thyroid (= hypothyroidsm) bezwichnet wird und zu folgen wie Headaches, Nausea, depression führen kann. In der kommenden Analyse wird die Aktivität des thyroxine biosythese pathways in thyroid cancers untersucht. [Thyroxine Deficiency? 17 Signs - ProgressiveHealth.com](https://www.progressivehealth.com/thyroxine-deficiency.htm)
Dadurch dass man weiß dass Thyroxine nicht vorhanden, kann man das ersetzen

the 3 histological types, was bringt uns das, die 3 Typen zu kennen??

2 Subtypen (differentiated (papilarry fulicular thca, weil die wie ursprünglihce thyroid cells ausshen) undifferentited, sehen einfach anders aus als andere thyroid cells, nicht klar abzugrenzen,

das andere bedeuted aus den

neue klassifikationen nur von PTC (BRAF like und RAS like) 80%s

## Hallmarks of cancer

Hallarks: was sind hallmarks? - Hallmarks sind Eigenschaften von TUmoren, die in jedem Tumor nachgewiesen werden können. Dazu gehören unter anderen die folgenden EIgenschaften: Resisting sell death, inducung angiogenesis, enabling replicative immortality, activating invasion and metastasis evading growth surpressors. Diese wurden zuerst von Hanahan und Weinberg 2011 veröffentlicht. Im laufe der Zeit kommen immer mehr hallmakrs dazu/es werden neue entdeckt. \[Quelle\]

Epigeentic profiles = auch epigenetische veränderungen werden in die Expressionsdate mit inebezogen, das wäre sehr sinnvol für die ANalyse, wird hier aber nicht beachetet

Was ist unsere Fragestellung?? und warum ist genau die so interessant?? und wie haben wir das erreicht=

## Computational tools

Um die Aktivität von Pathways zu vergleichen wurde eine

GSE methods um gene zu analysieren die bei RNA- seq analyse´siert wurden (fangen an mit einer ranked gene list =>x single enrichment score für jedes Geneset (2 genesets werden verglichen also zb normales und tumorgewebe)

#### Gene Set Variation Analysis

The Gene Set Variation Analysis (GSVA) is performed with the same intention as the GSEA, with the difference, that there is no reference expression data needed like in the GSEA. Because there was no expression data provided for comparison in the TCGA analysis, GSVA was used. There a various solutions to perform GSVA, one of them is to z-transform the provided expression data of the analysed tissue, so the mean over all patients of each gene is zero and the standard deviation is 1. The distribution of these z-transformed genes is like a t-distribution and can be used to calculate the log2FC and continue like in the GSEA.

#### Gene Set Enrichment Analysis 

To analyse and compare the activity of pathways a Gene Set Enrichment Analysis (GSEA) is performed. Therefore a reference is needed to compare the activity of certain pathways in tumor tissue with the activity in normal tissue.

First, the log2FC is calculated for each gene and ranked in a vector for each patient, beginning with the highest log2FC. This resulted in a MATRIX/DATAFRAME/LIST. A high log2FC implies, that the the gene is higher expressed in tumor tissue than in normal tissue. In the next step the activity of each pathway in each patient is calculated. Therefore the ranked vectors of each patient containing the log2FCs are used. The function checks for each gen if it is included in the analysed pathway or not. If it is included, the log2FC of that gene is added to the cumulative function, if it is not included, the log2FC is subtracted. This results in a cumulative function, that has a peak at a certain place. In this place of the ranked vector is also a gene located, the expression value of that gene is saved as the enrichment score of the analysed pathway and the patient belonging to the used vector.

GSEA was performed with the package xxx.

#### UMAP (**Uniform Manifold Approximation and Projection for Dimension Reduction)**

The UMAP is a method to reduce the dimension of a multidimensional data set. Compared to the PCA, the structure of the data in higher dimensions is maintained. Thereby the UMAP keeps the overall structure of the data set, therefore clusters are easier to identify.

The problem of the UMAP is, that although the overall structure is conserved, the distance between the individual points is not proportional to the real distance in the data set.

kleine KLumpen sind wahrscheinlcich auch im richtigen RAum zusammen

#### PCA (Principle component analysis)

Reduce the dimension of a given data set. The dimensions are summarized in principal components (PCs) which do not correlate. Because the PCs summarize the dimensions, the first PCs explain most of the variance of the data set and thereby can be selected to explain the data. Still, one has to keep in mind, that by reducing the dimensions, not all of the variance is explained and some of the information is lost in the process. The ideal number of PCs can be determined with an elbow-plot. In our analysis we use a PCA as a foundation for the UMAP, because the UMAP can not work with correlated dimensions. Furthermore it is used to detect the most important pathways, which explain most of the first PCs.

In the analysis, a PCA is performed for the pan cancer analysis on the TCGA gene expression data, to find similarities and differences in pathway activity for each tumor type. Furthermore a PCA is performed for the focused analysis of THCA and normal tissue.

### Jaccard index

The Jaccard index is the Intersection, divided by the union of two sets.

### Our Analysis

In the following 2 analyses are performed, a pan cancer analysis and a focused analysis about THCA.

#### Pan Cancer Analysis

For the pan cancer analysis 3 data sets are provided. One containing expression data of 60,000 genes in 10,000 tumor patients, another one with clinical annotations concerning those patients and one with hallmark pathways and their included genes. In the following analysis this data is cleaned by removing NAs, biotype filtering and low-variance filtering. After that a descriptive analysis is performed. Those two steps lead to the actual analysis, a gene set variation analysis to detect significantly altered pathways compared to the other pathways in tumor tissue. In the end a linear regression analysis is performed to predict pathway activity based on other pathways??? xxx Furthermore a neuronal network is built to improve prediction.

#### Focused analysis on THCA patients

Furthermore a analysis of THCA patients is performed. For this analysis a data set containing the gene expression of 60 patients in tumor an normal tissue and their clinical annotations. First the data is cleaned and described like the pan cancer data, to prepare the data for the gene set variation analysis, which is also performed for the THCA data in the bigger pan cancer data set, to confirm results from the smaller data set. In this analysis a linear regression analysis is performed to predict the activity of thyroxine biosynthesis. The results are also improved with a neuronal network.

## Cancer

You can cite one or multiple authors. One author [@kumar_multiple_2017\] and multiple authors \[@kumar_multiple_2017; @zavidij_single-cell_2020]. Write in **bold** or in *italic* or in both ***bolditalic***. You can also write inline code, e.g. `Seurat::RunUMAP`.

## LUAD

Some information @kumar_multiple_2017

## Computational Tools

### Gene Set Variation Analysis

### Gene Set Enrichment Analysis

### UMAP

### PCA

## Our Analysis

### Pan Cancer

### Focused Analysis

### Related Work


# Material and Methods

## Our data sets

For the analysis four data sets were provided.

The first data set is a Gene expression data frame. The Gene expression data frame contains 60,000 genes and and their expression in 10,000 patients. It is derived from The Cancer Genome Atlas (TCGA). The expression of the genes was obtained by RNA-seq, a sequencing method to evaluate the activity of the genes \ref{RNAseq}.

The second data frame contains 37 clinical annotations like Tumor type, age, gender, etc. concerning the 10,000 patients from the Gene expression data frame.

The third object contains 5 lists for the focused analysis, one for each tumor type (BRCA, KIRC, LUAD, PRAD, THCA). For the focused analysis the THCA data was used. The THCA data contains 3 data frames, each one with information about the same 60 patients. The first one with gene expression data from THCA tissue, the second one from normal tissue and the third one contains the clinical annotations like age and gender. Gene expression data was obtained by RNA-seq.

The fourth object contains 46 pathways involved in phenotypes partly included in the hallmarks of cancer and the genes involved in those pathways.

SIND DIE DATEN NORMALISIERT ODER ALS COUNTS?

## Metabolic pathway selection

Furthermore metabolic pathways had to be selected from the Molecular Signature Database (MSigDB) (QUELLE) to compare them with the given Hallmark-Pathways and identify more pathways, that are important for the development of cancer. Therefore it was important, that as many genes from the selected pathways as possible are also included in the provided Hallmark pathways. To identify the relevant pathways, the intersection of genes was calculated and the genes with an intersection of at least 99% were maintained for further analysis.

To avoid duplicates in between the metabolic pathways and between the Hallmark pathways and the metabolic pathways, the pathways were checked for duplicates with the Jaccard index. Pathways with a sum of Jaccard indices beyond the 1-sigma range were discarded.

## Preprocessing

### Deleting NAs

Deleting of NAs was done with the R-function na.omit(x).

### Low-variance filtering

Low variance filtering is performed to delete genes with a low variance from the data set. To calculate the variance the r-function var(x) is used and genes with a lower variance than a certain threshold value are removed.

For focused analysis the variance of each gene in tumor tissue was calculated and genes with a low variance were deleted from the data sets for tumor and normal tissue.

### Biotype filtering

For biotype filtering the biotype of each gene was determined (protein coding, RNA,...) and compared with the biotypes of other data sets. To allow an appropriate comparison of the expression data, only biotypes were kept, that are available in a big amount in all of the data sets. The biotype can be determined with the R-function checkbiotypes(x) from the package biomaRt.

#### Selection of metabolic pathways

da eine hohe jaccard summe eine hohe überschneidung mit anderen pathways bedeutet. In einer heatmap sind hohe Jacccard indices weiß bis rot gefährbt. Ein niedriger Jaccard index ist blau gefärbt.

The test for duplicates in between the selected metabolic pathways and between the hallmark pathways and the metabolic pathways the Jaccard index and its sum were calculated.

There were a few duplicates between the metabolic and Hallmark pathways. Those metabolic pathways with a great sum of Jaccard indices were discarded. The success of cleaning was checked with another heatmap and new Jaccard indices. There are only a few elements with another color than blue. It can be assumed, that the selection of relevant pathways was successful. The number of metabolic pathways could be reduced from xxx to 600.

## Descriptive analysis

### Mean-variance plot

In a mean-variance plot the variance is plotted against the mean of expression values of the single genes. Therefore, the variance and mean were calculated by the R-function var(x) and mean(x).

### Violin plot

To check the distribution of a data set and compare it with other data sets violin plots are used. Violin plots are tilted and mirrored density plots of gene expression values. The y-axis shows the gene expression value and the x-axis shows the amount of genes with a certain gene expression value.

#### Volcano plot

A volcano plot is used to identify significantly differentially expressed genes. In the following analysis a volcano plot is used to identify which genes are significantly diferentially expressed in the analysed THCA tissue, compared to the analysed normal tissue. Therefore the mean of each gene is calculated for normal and THCA tissue and used for the calculation of the Log2-Foldchange (Log2FC) in the following way:

$$
log2FC = mean(normal tissue) - mean(tumor tissue)
$$
In the next step, a two-sided t test was performed to determine the significance of a difference in expression.

To avoid the accumulation of type 1 errors, a bonferroni correction was performed. n is the number of genes in the cleaned data set for focused analysis:

$$
A \alpha = \frac{0.025}{n}
$$

In the volcano plot the -log10 of the calculated p values is plotted against die Log2FC. Genes with a a lower p-value than the corrected alpha-value are significantly differentially expressed. If the Log2FC is additionally higher than 0.1, the genes are significantly over expressed in tumor tissue, if the Log2FC is higher lower than -0.1, the genes are significantly under expressed in tumor tissue.

## Comparing of Pathways

After reducing the number of Genes of the Gene Expression data frame and the THCA data by data cleaning, a descriptive analysis was performed with a Mean-variance plot and five violin plots of the TGCA data frame. The descriptive analysis of the THCA data was performed with a volcano plot and the distribution of the Tumor-specific data was displayed with violin plots.

### PCA

The package xxx is used to perform the PCA. Therefore the data obtained from the GSEA was used. After performing the PCA, the results were plotted to visualize the different clusters.

The PCA was performed for pathway and gene activity. For analysis of the gen activity the package xxx was used.

Dazu wurde noch analysiert, wie die Pathways auf die PCs verteilt sind.

## TCGA data

What kind of data do we have? ## Used Packages

show a table!

`#``{r child = "chapter/results.Rmd"}
\end{verbatim}

`\texttt{\#}\{r child = ``chapter/discussion.Rmd''\}

\begin{verbatim}
/ref{RNAseq}
/ref{histological_types}
[@RNAseq]
# References

::: {#refs}
:::


# Appendix

## Plots
hello
## Code
world


```r
#createn einer liste mit allen patienten in dfs sortiert nach krebs
cancers = list();cancers = vector('list',length(table(tcga_anno$cancer_type_abbreviation)))
names(cancers) = names(table(tcga_anno$cancer_type_abbreviation))
i=1
for (i in 1:length(cancers)){
  cancers[[i]] = tcga_exp_cleaned[,tcga_anno$cancer_type_abbreviation == names(cancers)[i]]
}
#function die einen krebstypen df und genesets als input nimmt und ein df mit pvalues ausgibt
enrichment = function(expressiondata, genesets = genesets_ids){
  ESmatrix = sapply(genesets, FUN = function(x){
    ins = na.omit(match(x,rownames(expressiondata)))#indices der gene im aktuellen set
    outs = -ins#indices der gene nicht im aktuellen set
    #gibt einen vektor der für jeden patienten den pval für das aktuelle gene enthält
    res = NULL
    for (i in 1:ncol(expressiondata)){#testet für jeden patienten
      res[i] = wilcox.test(expressiondata[ins,i],expressiondata[outs,i],'two.sided')$p.value
    }
    return(res)
  })
  row.names(ESmatrix) = colnames(expressiondata); return(ESmatrix)
}
pvalueslist = lapply(cancers, enrichment)#für die tests für jeden krebstypen durch
\end{verbatim}

\begin{Shaded}
\begin{Highlighting}[]
\NormalTok{get\_top10pathways\_from\_pvalues }\OtherTok{=} \ControlFlowTok{function}\NormalTok{(df\_p\_values, length\_genesets) \{}
  
  \FunctionTok{require}\NormalTok{(ggplot2)}
  
\NormalTok{  results }\OtherTok{\textless{}{-}} \FunctionTok{list}\NormalTok{()}
    
\NormalTok{  df\_p\_values\_log10 }\OtherTok{\textless{}{-}} \SpecialCharTok{{-}}\FunctionTok{log10}\NormalTok{(}\FunctionTok{as.data.frame}\NormalTok{(df\_p\_values))}
    
\NormalTok{  mean\_pathway }\OtherTok{\textless{}{-}} \FunctionTok{as.data.frame}\NormalTok{(}\FunctionTok{apply}\NormalTok{(df\_p\_values\_log10, }\DecValTok{1}\NormalTok{, mean))}
  \FunctionTok{rownames}\NormalTok{(mean\_pathway) }\OtherTok{\textless{}{-}} \FunctionTok{rownames}\NormalTok{(df\_p\_values\_log10)}
  
\NormalTok{  ordered\_score }\OtherTok{\textless{}{-}}\NormalTok{ mean\_pathway[}\FunctionTok{order}\NormalTok{(}\SpecialCharTok{{-}}\NormalTok{mean\_pathway[ ,}\DecValTok{1}\NormalTok{]), }\DecValTok{1}\NormalTok{]}
\NormalTok{  top\_10 }\OtherTok{\textless{}{-}} \FunctionTok{data.frame}\NormalTok{(ordered\_score[}\DecValTok{1}\SpecialCharTok{:}\DecValTok{10}\NormalTok{])}
  \FunctionTok{colnames}\NormalTok{(top\_10) }\OtherTok{\textless{}{-}} \StringTok{"mean\_pathway"}
  
\NormalTok{  ordered\_names }\OtherTok{\textless{}{-}} \FunctionTok{order}\NormalTok{(}\SpecialCharTok{{-}}\NormalTok{mean\_pathway[ ,}\DecValTok{1}\NormalTok{])}
\NormalTok{  top\_10\_names }\OtherTok{\textless{}{-}}\NormalTok{ ordered\_names[}\DecValTok{1}\SpecialCharTok{:}\DecValTok{10}\NormalTok{]}
\NormalTok{  top\_10}\SpecialCharTok{$}\NormalTok{pathway\_names }\OtherTok{\textless{}{-}} \FunctionTok{row.names}\NormalTok{(mean\_pathway)[top\_10\_names]}
  
\NormalTok{  results[[}\DecValTok{1}\NormalTok{]] }\OtherTok{\textless{}{-}}\NormalTok{ top\_10}
  
\NormalTok{  results[[}\DecValTok{2}\NormalTok{]] }\OtherTok{\textless{}{-}} \FunctionTok{ggplot}\NormalTok{(}\AttributeTok{data =}\NormalTok{ top\_10, }\FunctionTok{aes}\NormalTok{(}\AttributeTok{x =}\NormalTok{ mean\_pathway, }\AttributeTok{y =} \FunctionTok{reorder}\NormalTok{(pathway\_names, mean\_pathway)))}\SpecialCharTok{+}
    \FunctionTok{geom\_bar}\NormalTok{(}\AttributeTok{stat =} \StringTok{"identity"}\NormalTok{)}\SpecialCharTok{+}
    \FunctionTok{coord\_cartesian}\NormalTok{(}\AttributeTok{xlim =}\FunctionTok{c}\NormalTok{(}\DecValTok{3}\NormalTok{, }\FloatTok{3.75}\NormalTok{))}\SpecialCharTok{+}
    \FunctionTok{labs}\NormalTok{(}\AttributeTok{title =} \FunctionTok{names}\NormalTok{(df\_p\_values),}
         \AttributeTok{x =} \StringTok{"mean p{-}value pathway"}\NormalTok{,}
         \AttributeTok{y =} \StringTok{"pathway name"}\NormalTok{)}
  
\NormalTok{  pathway\_size }\OtherTok{\textless{}{-}} \FunctionTok{order}\NormalTok{(}\SpecialCharTok{{-}}\NormalTok{mean\_pathway[ ,}\DecValTok{1}\NormalTok{])}
\NormalTok{  top\_10\_size }\OtherTok{\textless{}{-}}\NormalTok{ pathway\_size[}\DecValTok{1}\SpecialCharTok{:}\DecValTok{10}\NormalTok{]}
\NormalTok{  top\_10}\SpecialCharTok{$}\NormalTok{pathway\_size }\OtherTok{\textless{}{-}}\NormalTok{ length\_genesets[top\_10\_size]}
  
\NormalTok{  results[[}\DecValTok{3}\NormalTok{]] }\OtherTok{\textless{}{-}} \FunctionTok{ggplot}\NormalTok{(}\AttributeTok{data =}\NormalTok{ top\_10, }\FunctionTok{aes}\NormalTok{(}\AttributeTok{x =}\NormalTok{ mean\_pathway, }\AttributeTok{y =} \FunctionTok{reorder}\NormalTok{(pathway\_names,}
\NormalTok{                                                                          mean\_pathway)))}\SpecialCharTok{+}
    \FunctionTok{geom\_point}\NormalTok{(}\FunctionTok{aes}\NormalTok{(}\AttributeTok{size =}\NormalTok{ pathway\_size))}\SpecialCharTok{+}
    \FunctionTok{labs}\NormalTok{(}\AttributeTok{title =} \FunctionTok{names}\NormalTok{(df\_p\_values),}
         \AttributeTok{x =} \StringTok{"mean p{-}value pathway"}\NormalTok{,}
         \AttributeTok{y =} \StringTok{"pathway name"}\NormalTok{)}
  
  \FunctionTok{return}\NormalTok{(results)}
\NormalTok{\}}
\end{Highlighting}
\end{Shaded}


\end{document}
