---
title: "Schildkrötenkrebs"
author: "Jakob Then, Anna Lange, Maren Schneider, David Matuschek"
date: "18 07 2022"
output:
  bookdown::html_document2:
    df_print: paged
    highlight: pygments
    number_sections: yes
    theme: journal
    toc: yes
---

```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
```

# Introduction

2019 starben 230,242 Menschen in Deutschland an Krebs \[Quelle\] \[[Krebsrate und Krebs-Sterberate in Deutschland (krebsinformationsdienst.de)](https://www.krebsinformationsdienst.de/tumorarten/grundlagen/krebsstatistiken.php)\]. Um Tumore zu erkennen und eine besserer Behandlung zu finden, ist die Entwicklung neuer Behandlungsmethoden von hoher Wichtigkeit. Dazu ist es essenziell die mutationstechnischen uRsachen für Krebsentwicklung zu identifizieren. Dafür können transcriptomic profiling methods wie RNA-seq verendet werden.

The provided data in the following analysis originates from a transcriptomic profiling methods like RNA-seq. Transcriptomic profiling sequences all the RNA that has been generated by transcription of the cells DNA. The difference to sequencing of DNA is, that it only sequences those genes, that are going to be expressed in that cell.

#### RNA-seq

RNA-seq is performed by cleaning of RNA, fragmentation, translation of RNA to cDNA, sequencing of cNDA and comparing with the reference genome. The advantage of RNA-seq is that it includes information about gene expression, that is especially important in the analysis of tumors such as epigenetic changes (e.g. epigenetic gene silencing) or fusion proteins.

The results from RNA-seq used for the analysis originate from the cancer genome atlas (TCGA)

## Thyroid carcinoma (THCA)

THCA incidence increased dramatically over the past few years (<https://jamanetwork.com/data/journals/intemed/936342/m_ied170005f1.png>) REFERENCE, deswegen schauen wir uns die Gene, die THCA verursachen im unserer Analyse näher an. Allerdings sind nur 1% der deutschen Tumoren THCA.

Thyroid cancers werden in verschiedene Typen aufgeteilt, der häufigste Typ ist papillary thyroid cancer (PTC) with 80% of total Thyroid cancers. The most common mutation in PTC is the V600E-Mutation of the RAF-Kinase, what causes a constant activation and by intracellular signaling it promotes tumor cell growth and thereby growth of the tumor.

Normally the Thyroid gland controls the body temperature and the metabolism.

the 3 histological types, was bringt uns das, die 3 Typen zu kennen??

## Hallmarks of cancer

Hallarks: was sind hallmarks? - Hallmarks sind Eigenschaften von TUmoren, die in jedem Tumor nachgewiesen werden können. Dazu gehören unter anderen die folgenden EIgenschaften: Resisting sell death, inducung angiogenesis, enabling replicative immortality, activating invasion and metastasis evading growth surpressors. Diese wurden zuerst von Hanahan und Weinberg 2011 veröffentlicht. Im laufe der Zeit kommen immer mehr hallmakrs dazu/es werden neue entdeckt. \[Quelle\]

Epigeentic profiles = auch epigenetische veränderungen werden in die Expressionsdate mit inebezogen, das wäre sehr sinnvol für die ANalyse, wird hier aber nicht beachetet

Was ist unsere Fragestellung?? und warum ist genau die so interessant?? und wie haben wir das erreicht=

## Computational tools

GSE methods um gene zu analysieren die bei RNA- seq analyse´siert wurden (fangen an mit einer ranked gene list =>x single enrichment score für jedes Geneset (2 genesets werden verglichen also zb normales und tumorgewebe)

#### Gene Set Variation Analysis (GSVA)

The GSVA is performed with the same intention as the GSEA, with the difference, that there is no reference expression data needed like in the GSEA. Because there was no expression data provided for comparison in the TCGA analysis, GSVA was used. There a various solutions to perform GSVA, one of them is to z-transform the provided expression data of the analysed tissue, so the mean over all patients of each gene is zero and the standard deviation is 1. The distribution of these z-transformed genes is like a t-distribution and can be used to calculate the log2FC and continue like in the GSEA.

Eine Form von Gene set ebrichment analysis

schätzt die veränderung Pathway aktivität eine Population, unsupervised = kleine Verönderung der ptahwayaktivität werden erkannt

Analyse muss für froße, heterogene populationen mit komplexen phänotypischen Merkmalen möglich sein

=> Lösung GSVA: nicht-parametrisch, unsupervised

1.  input: gene expression matrix mit rna seq counts in unserem Fall
2.  ERstellen der Gene expression level statistic von Gen i in einem sammple durch Berechnen der kumulativen Verteilungsfunktion von allen Genexpressionsdaten
3.  Ranke der gene expression statistics
4.  Kolmogorov-Smirnov-like rank statistic wird für jedes gene set berechnet

#### Gene Set Enrichment Analysis (GSEA)

To analyse and compare the activity of pathways a GSEA is performed. Therefore a reference is needed to compare the activity of certain pathways in tumor tissue with the activity in normal tissue. Therefore the log2FC is calculated for each gene and ranked in a vector for each patient, beginning with the highest log2FC. This resulted in a MATRIX/DATAFRAME/LIST. A high log2FC implies, that the the gene is higher expressed in tumor tissue than in normal tissue. In the next step the activity of each pathway in each patient is calculated. Therefore the ranked vectors of each patient containing the log2FCs are used. The function checks for each gen if it is included in the analysed pathway or not. If it is included, the log2FC of that gene is added to the cumulative function, if it is not included, the log2FC is subtracted. This results in a cumulative function, that has a peak at a certain place. In this place of the ranked vector is also a gene located, the expression value of that gene is saved as the enrichment score of the analysed pathway and the patient belonging to the used vector.

GSEA was performed with the package xxx.

#### UMAP (**Uniform Manifold Approximation and Projection for Dimension Reduction)**

The UMAP is a method to reduce the dimension of a multidimensional data set. Compared to the PCA, the structure of the data in higher dimensions is maintained. Thereby the UMAP keeps the overall structure of the data set, therefore clusters are easier to identify.

The problem of the UMAP is, that although the overall structure is conserved, the distance between the individual points is not proportional to the real distance in the data set.

kleine KLumpen sind wahrscheinlcich auch im richtigen RAum zusammen

#### Principle component analysis

The Principle component analysis (PCA) is used to reduce the dimension of a given data set. The dimensions are summarized in principal components (PCs) which do not correlate. Because the PCs summarize the dimensions, the first PCs explain most of the variance of the data set and thereby can be selected to explain the data. Still, one has to keep in mind, that by reducing the dimensions, not all of the variance is explained and some of the information is lost in the process.

The ideal number of PCs can be determined with an elbow-plot. In our analysis we use a PCA as a foundation for the UMAP, because the UMAP can not work with correlated dimensions. Furthermore it is used to detect the most important pathways, which explain most of the first PCs.

In the analysis, a PCA is performed for the pan cancer analysis on the TCGA gene expression data, to find similarities and differences in pathway activity for each tumor type. Furthermore a PCA is performed for the focused analysis of THCA and normal tissue.

### Jaccard index

The Jaccard index is the Intersection, divided by the union of two sets. WARUM MACHEN WIR DAS

# Material and Methods

## Our data sets

For the analysis 4 data sets were provided.

The first data set is a Gene expression data frame. The Gene expression data frame contains 60,000 genes and and their expression in 10,000 patients, and derived from The Cancer Genome Atlas (TCGA). The expression of those genes was obtained by RNA-seq.

The second data frame contains 37 clinical annotations like Tumor type, age, gender, etc. concerning the 10,000 patients from the Gene expression data frame.

The third object contains 5 lists for the focused analysis, one for each tumor type (BRCA, KIRC, LUAD, PRAD, THCA). For the focused analysis the THCA data was used. The THCA data contains 3 data frames, each one with information about the same 60 patients. The first one with gene expression data from THCA tissue, the second one from normal tissue and the third one contains the clinical annotations like age and gender. Gene expression data was obtained by RNA-seq.

The fourth object contains 46 pathways involved in phenotypes partly included in the hallmarks of cancer and the genes involved in those pathways.

Log2(TMP) = Transcripts per million

## Metabolic pathway selection

Furthermore metabolic pathways had to be selected from the Molecular Signature Database (MSigDB) (QUELLE) to compare them with the given Hallmark-Pathways and identify more pathways, that are important for the development of cancer. Therefore it was important, that as many genes from the selected pathways as possible are also included in the provided Hallmark pathways. To identify the relevant pathways, the intersection of genes was calculated and the genes with an intersection of at least 99% were maintained for further analysis.

```{r showhistcanonical, echo=FALSE, eval=TRUE, out.width = '100%', fig.align = "center", fig.cap="**Mean gene expression split by tumour type.** Log2-transformed gene expression values, given in transcripts per million (TPM), were averaged per tumour type and visualised in a violin plot."}

knitr::include_graphics("tcga_exp_biotypes.png")

```

To avoid duplicates in between the metabolic pathways and between the Hallmark pathways and the metabolic pathways, the pathways were checked for duplicates with the Jaccard index. Pathways with a sum of Jaccard indices beyond the 1-sigma range were discarded.

## Preprocessing

### Deleting NAs

Deleting of NAs was done with the R-function na.omit(x).

### Low-variance filtering

Low variance filtering is performed to delete genes with a low variance from the data set. To calculate the variance the r-function var(x) is used and genes with a lower variance than a certain threshold value are removed.

For focused analysis the variance of each gene in tumor tissue was calculated and genes with a low variance were deleted from the data sets for tumor and normal tissue.

### Biotype filtering

For biotype filtering the biotype of each gene was determined (protein coding, RNA,...) and compared with the biotypes of other data sets. To allow an appropriate comparison of the expression data, only biotypes were kept, that are available in a big amount in all of the data sets. The biotype can be determined with the R-function checkbiotypes(x) from the package biomaRt.

#### Selection of metabolic pathways

da eine hohe jaccard summe eine hohe überschneidung mit anderen pathways bedeutet. In einer heatmap sind hohe Jacccard indices weiß bis rot gefährbt. Ein niedriger Jaccard index ist blau gefärbt.

The test for duplicates in between the selected metabolic pathways and between the hallmark pathways and the metabolic pathways the Jaccard index and its sum were calculated.

There were a few duplicates between the metabolic and Hallmark pathways. Those metabolic pathways with a great sum of Jaccard indices were discarded. The success of cleaning was checked with another heatmap and new Jaccard indices. There are only a few elements with another color than blue. It can be assumed, that the selection of relevant pathways was successful. The number of metabolic pathways could be reduced from xxx to 600.

## Descriptive analysis

### Mean-variance plot

In a mean-variance plot the variance is plotted over the mean of expression values of the single genes across all patients. Thus, the variance and mean were calculated by the R-function var(x) and mean(x). This is done to determine genes, which differ a lot in their expression levels across all patients. The plot is created with the package citation(ggplot2). 

### Violin plot

To check the distribution of a data set and compare it with other data sets violin plots are used. Based on how similar the violinplots are, it can be implied if the data is normalized. Violin plots are tilted and mirrored density plots of gene expression values. The y-axis shows the gene expression value and the x-axis shows the amount of genes with a certain gene expression value.

#### Volcano plot

A volcano plot is used to identify significantly differentially expressed genes. In the following analysis a volcano plot is used to identify which genes are significantly diferentially expressed in the analysed THCA tissue, compared to the analysed normal tissue. Therefore the mean of each gene is calculated for normal and THCA tissue and used for the calculation of the Log2-Foldchange (Log2FC) in the following wIn the next step, a two-sided t test was performed to determine the significance of a difference in expression.

To avoid the accumulation of type 1 errors, a bonferroni correction was performed. The variable n is the number of genes in the cleaned data set for focused analysis:

$$
 \alpha = \frac{0.025}{n}
$$

In the volcano plot the -log10 of the calculated p values is plotted against die Log2FC. Genes with a a lower p-value than the corrected alpha-value are significantly differentially expressed. If the Log2FC is additionally higher than 0.1, the genes are significantly over expressed in tumor tissue, if the Log2FC is higher lower than -0.1, the genes are significantly under expressed in tumor tissue.

## Comparing of Pathways

After reducing the number of Genes of the Gene Expression data frame and the THCA data by data cleaning, a descriptive analysis was performed with a Mean-variance plot and five violin plots of the TGCA data frame. The descriptive analysis of the THCA data was performed with a volcano plot and the distribution of the Tumor-specific data was displayed with violin plots.

### PCA

The package xxx is used to perform the PCA. Therefore the data obtained from the GSEA was used. After performing the PCA, the results were plotted to visualize the different clusters.

The PCA was performed for pathway and gene activity. For analysis of the gen activity the package xxx was used.

Dazu wurde noch analysiert, wie die Pathways auf die PCs verteilt sind.

# Results

## Preprocessing

#### deleting NAs

Deletion of NAs was applied to the 2 gene expression data frames. Because the dimension of our data frames did not change during this process it was assumed, that there were no NAs in the data sets.

#### Low-variance filtering

The goal of the analysis was to itentify the genes that show a significantly different expression in certain tumor types (Pan cancer analysis) or in comparison from normal and tumor tissue (THCA Analysis). Therefore genes with a similar expression in all patients are not relevant.

The logarithm of the obtained values were displayed in a histogram shown in Figure xxx. The genes with a lower variance than -1 were omitted. The number of genes was reduced from 60,000 to 19,000 genes.

The low-variance filtering of the THCA dataset was done in a similar way. Genes with a lower variance than -1.25 were deleted. This resulted in a reduction from about 20,000 genes to 15,000 genes in the data frames for normal and tumor tissue.

#### Biotype filtering

The biotype of the genes from the selected metabolic pathways, the genes of the hallmark pathways and the genes of the gene expression matrix was determined, to keep those genes with the same bio type. Because the most genes are protein-coding, only protein-coding genes were maintained.

```{r showbiotypeshallmarks, echo=FALSE, eval=TRUE, out.width = '20%', fig.align = "center", fig.cap="**Biotypes of Hallamrk genes**"}

knitr::include_graphics("tcga_exp_biotypes.png")

```

```{r showbiotypesexp, echo=FALSE, eval=TRUE, out.width = '20%', fig.align = "center", fig.cap="**Biotypes of Gene expression Data frame**"}

knitr::include_graphics("tcga_exp_biotypes.png")

```

```{r showbiotypesmetabolic, echo=FALSE, eval=TRUE, out.width = '20%', fig.align = "center", fig.cap="**Biotypes of metabolic pathways**"}

knitr::include_graphics("tcga_exp_biotypes.png")

```

### Descriptive analysis

#### Mean-variance Plot

To determine the genes from the TCGA expression data with a high variance, the variance was plotted over the mean (Figure \@ref(fig:showmeanvariance)). Additionally those genes with a variance higher than 33 were labeled with their EnsembleID. The distribution of genes in this plot shows that the highly variant genes are around a mean expression level of 0. The plot also shows, that very few genes are at a low mean expression level like -10 or at a ver high mean expression level like 10. Most genes are expressed across all at a mean expression level around 0. Via this plot we were able to determine which genes differ a lot in their expression level across all cancer patients.

```{r showmeanvariance, echo=FALSE, eval=TRUE, out.width='30%', fig.align='center', fig.cap="Mean-variance plot of cleaned TCGA expression data. Y-axis shows variance of a genes expression, x-axis shows mean of a genes expression"}

knitr::include_graphics("Variance_over mean_cleaned_matrix.png")

```

![](http://127.0.0.1:26567/chunk_output/3DCC3289CACBD6C3/219AFA97/cc46yulwlpcl4/000003.png?fixed_size=1)

#### Violin Plots

Five violin plots were created for five different tumor types with the gene expression data from the TCGA. This was done to compare the distribution of the data of five different tumor types (Figure \@ref(fig:showviolinplots)). The white point in the middle of each plot shows the median. The median gene expression is at around 2.5 across the five cancer types. The upper and lowe a border of the box plot, which show the 25% and 75% quantile are at rougly the same level across all five tumor types. The distribution flattens at around the very high and very low expression leves, because only a few genes are expressed at a very high or very low level. One can see, that the the distributions of all five tumor types are similar. This implies, that the other 28 tumor types of this data set are distributed in a similar way and the data is normalized. 

```{r showviolinplots, echo=FALSE, eval=TRUE, out.width='30%', fig.align='center', fig.cap="Mean-variance plot of cleaned TCGA expression data"}

knitr::include_graphics("Vioplots cancer types.png")

```

#### Volcano plot

For the construction of the volcano plot the data for THCA from the data set for the focused analysis was used. We used the expression data from tumor tissue and compared it to the data from normal tissue. This was done to determine those genes, which are up- or downregulated in Thyroid carcinoma (Figure \@ref(fig:showvolcanoplot)). The significance level was determinated at 1.755e-06. We then colored those genes, which are not up- or down-regulated at a significant level green. Based on their foldchange we were able to determine those genes, which were up-regulated in THCA tumor tissue (blue) and, which were down-regulated in tumor tissue (red). The genes with a very low p-value differ the most from tumor vs. normal tissue and were annotated with their name. 

```{r showvolcanoplot, echo=FALSE, eval=TRUE, out.width='30%', fig.align='center', fig.cap="Volcano plot of THCA expression data"}

knitr::include_graphics("Volcanoplot.png")

```

#### GSEA

The results of the GSEA for THCA tissue can be seen in Figure \@ref(fig:GSEAHeat)

```{r GSEAHeat, echo=FALSE, eval=TRUE, out.width='50%', fig.align='center', fig.cap="GSEA performed on the THCA expression data, annotated with Pathway type, histoligical type and cluster."}

knitr::include_graphics("THCA GSEA Heatmap fertig.png")
```

Horizontal sind die einzelnen Patienten angeordnet, vertikal die pathways, die Heatmap stellt die stärke der Expression der einzelnen pathways in den einzelnen Patienten dar. Rot markierte pathways sind überexprimiert, blaue sind unterexprimiert. Die Achsen wurden mit dem pathway typ, dem hiytological typ und dem cluster annotiert. Es bilden sich innnerhalb der Patienten 3 Cluster, die auf Gemiensamkeiten in der PAthwayaktivität zurückgeführt werden können. Vielleicht hängt die Clusterbildung mit dem histological type zusammen, da eindeutig erkennbar ist, dass in Cluster 3 nur der classical type vorhanden ist, während in Cluster 2 vermehrt Tall cell type auftritt und Folicular type vermehr in Cluster 1 auftritt.

### pan cancer für THCA

Um die Clusterbildung zu bestätigen wurde die gleiche Analyse für die THCA daten aus dem großen gene expression dataframe durchgeführt. Auch in dieser Analyse formten sich 3 Cluster, die in zum Teil auf die hiytological types zurückzuführen sind. xxx

#### GSVA

xxx Was wir aus der Heatmap sehen???

```{r GSVAHeat, echo=FALSE, eval=TRUE, out.width='30%', fig.align='center', fig.cap="PCA of TCGA expression data, colored by tumor type"}

knitr::include_graphics("GSVA Heatmap fertig.png")
```

#### PCA

Die PCA wurde anhand der Pathway Aktivitäten durchgeführt und reduziert auf die ersten 2 PCs in Figure \@ref(fig:PCAPanType) dargestellt. Es sind ansatzweise cluster, aber keine eindeutigen Strukturen zu erkennen. Die zusammensetzung der PCs und die Erklärung der AVrianz durch die einzelnen VAriablen kann Figure \@ref(fig:) entnommen werden. BESCHREIBUNG DER PCA UND DER VARIANZVERTEILUNG. Der Plot wurde auch nach Form des Tumors angefärbt und in Figure \@ref(fig:PCAPanForm) dargestellt. Hier ist eine klarere Struktur zu erkennen als bei Anfärbung nach Tumortyp. Daraus lässt sich schließen, dass zumindest die PC1 und PC2 inkludierten Pathways mit der Form des Tumors zusammenhängen.

Um die Ergebnisse zu überprüfen wurde die PCA auch auf die Genaktivität angewendet, es war kein Unterschied zu erkennen.

```{r PCAPanType, echo=FALSE, eval=TRUE, out.width='30%', fig.align='center', fig.cap="PCA of TCGA expression data, colored by tumor type"}

knitr::include_graphics("Pan Cancer PCA PC1und2.png")
```

```{r PCAPanForm, echo=FALSE, eval=TRUE, out.width='30%', fig.align='center', fig.cap="PCA of TCGA expression data, colored by form of tumor"}

knitr::include_graphics("Pan Cancer PCA PC1und2 cancer form.png")
```

#### UMAP

DA durch die PCA keine eindeutigen Cluster erkennbar wurden, wurden die Ergebnisse der PCA genutzt, um eine UMAP zu erstellen. Die Ergebnisse sind in Figure \@ref(fig:UMAPPanType) zu sehen. Hier sind eindeutige Cluser zu erkennen, auch wenn die Abstände zwischen den Clustern nicht proportional zu den wahren Unterschieden sind, kann daraus geschlossen werden, dass die einzelnen Tumortypen charakteristische Pathwayaktivitäten haben, wie zum Beispiel KIRC. Viele Tumortypen haben aber auch ähnliche Pathwayaktivitäten wie zum Beispiel GBM und SARC.

Um die Ursache dafür zu finden wurde auch hier die UMAP nach Form des Tumors markiert/gefärbt. Die Ergebnisse sind in Figure \@ref(fig:UMAPPanForm) zu sehen. Es ist eindeutig zu erkennen, dass die Pathwayaktivität auch mit der Form des Tumors zusammenhängt. Zum Beispiel sind Glioblastome eindeutig von Adenocarcinomen getrennt. Allerdings ist auch hier das große Cluster in der Mitte nicht eindeutig einer Tumorform zuzuordnen.

```{r UMAPPanType, echo=FALSE, eval=TRUE, out.width='50%', fig.align='center', fig.cap="UMAP of TCGA expression data, colored by tumor type"}

knitr::include_graphics("Pan Cancer UMAP.png")
```

```{r UMAPPanForm, echo=FALSE, eval=TRUE, out.width='50%', fig.align='center', fig.cap="UMAP of TCGA expression data, colored by form of the tumor"}

knitr::include_graphics("Pan Cancer UMAP cancer form.png")
```

#### Regression analysis

### Focused analsis

#### PCA

PCA was also performed for focused analysis, um Untergruppen, die mit der Pathwayaktivität oder Genaktivität zusammenhängen, innerhalb der Thyroid tumors zu finden. Dafür wurden zuerst die Ergebnisse der GSEA verwendet. Der Versuch Cluster zu finden gestaltete sich hier leider schwerer, da weder durch Betrachtung des Thyroid Tumortyps, noch durch Betrachtung der Stage ein eindeutiger Zusammenhang erkannt werden konnte.
