---
editor_options:
  markdown:
    wrap: 72
output:
  pdf_document:
    df_print: paged
---

# Materials and Methods

For the means of this project two separate analysis are performed: a
pan-cancer analysis focusing on differences between cancer types and a
focused analysis investigating THCA.

## Description of the underlying data

For the analysis four data sets were provided. For pan-cancer analysis a
gene expression data frame with normalized and log2 transformed bulk
RNA-seq expression data for 60,489 genes in 9741 patients with 33
different forms of cancer was used. The data was derived from The Cancer
Genome Atlas (TCGA). Complementing the TCGA expression data is an
annotation dataframe with 37 clinical annotations regarding tumor type,
tumor stage, gender, age, etc. for all patients.

The third object is a list containing five lists for the focused
analysis, one list for each tumor type (BRCA, KIRC, LUAD, PRAD, THCA).
For our focused analysis, the only the THCA data were used. The THCA
list consists of three data frames: The first two contain normalized and
log2 transformed bulk RNA-seq expression data for 19,624 genes in 59
THCA patients for carcinogenic and homeostatic tissue. The third
dataframe complementes the data with the respective clinical
annotations.

The last object contains 46 pathways associated with the hallmarks of
cancer in form of a list of string vectors.

## Metabolic pathway selection

To perform enrichment analysis later on, 6366 canonical pathways were
selected from the Molecular Signatures Database (MSigDB)\[@msigdb\] with
the `msigdbr::msigdbr()` function. As not to introduce a bias during
enrichment analysis, the similarity of MSigDB pathways among themselves
as well as with the hallmark pathways was computed with the Jaccard
index. Pathways with a Jaccard index greater than the 1$\sigma$ range
were discarded.

## Preprocessing of expression data

### Data cleaning

All expression data were checked for missing values with the `na.omit()`
function. Subsequently, low variance filtering was performed for TCGA
and THCA tumor expression data. The variances of expression were
computed for every gene across all samples and then, genes with
variances blow a threshold were discarded to reduce dimensionality.

### Biotype filtering

Next, biotype filtering was performed for pan-cancer and THCA expression
data to reduce dimensionality further. Only genes sharing biotypes with
the hallmark pathways were kept for the the following analysis. The
biotypes of the genes were retrieved using the `biomart::getBM()`
function from the biomaRt package [@biomart]. To allow for an
appropriate comparison within all pathways, only MSigDB pathways where
over 99% of their respective genes were present in the filtered
expression data were selected as final pathways.

## Methods for descriptive analysis

### Mean-variance plot

In a mean-variance plot the variance is plotted over the mean of
expression values of single genes across all patients. Thus, the
variance and mean were calculated for each gene in the THCA expression
data. The final plot was created with the package ggplot2 \ref{ggplot2}.

### KANN RAUS m.M.n Violin plot

To check the distribution of a data set and compare it with other data
sets violin plots are used. Based on how similar the violin plots are,
it can be implied that the data is normalized. Violin plots are tilted
and mirrored density plots of gene expression values. The y-axis shows
the gene expression value and the x-axis shows the amount of genes with
a certain gene expression value.

### Jaccard-Index

The Jaccard-Index is a method to describe the similarity between two
quantities. To compute it, the intersection of all gene ENSEMBL-IDs from
two compared pathways was divided by their union. We used this method to
determine the degree in which pathways are similar to each other.

### Volcano plot

A volcano plot is used to identify genes displaying significantly
different expression in carcinogenic versus homeostatic tissues. First,
the log2 fold change (Log2FC) is calculated for each gene across all
samples in the THCA expression data in the following way:

$$
log2FC = mean(normal tissue) - mean(tumor tissue)
$$Next, a two-sided t-test was performed with the `t.test()` function to
determine the significance of a difference in expression. To avoid the
accumulation of type one errors, a Bonferroni correction was performed.
n is the number of genes in the cleaned data set for focused analysis:

$$
\alpha = \frac{0.025}{n}
$$

In the volcano plot the -log10 of the calculated p-values is plotted
against die Log2FC. Genes with a lower p-value than the corrected
significance level $\alpha$ are significantly differently expressed. If
the Log2FC is additionally positive, the genes are significantly
overexpressed in tumor tissue, if the Log2FC is negative, the genes are
significantly underexpressed in tumor tissue.

## Dimension reduction and pathway enrichment analysis

The GSEA was used to identify enriched pathways in THCA tumor tissue.
Here, GSEA was performed with the package "fgsea" [@fgsea]. First, the
expression values were ranked decreasingly by log2FC for every patient.
Log2FC was chosen as the ranking metric as it easy to compute and shows
a high sensitivity.

xxx Quelle: Ranking metrics in gene set enrichment analysis: do they
matter?

Secondly, using the ranked log2FC vectors, the enrichment score of each
pathway was calculated for each patient with the `fgseamultilevel()`
function.

### Geneset Variation Analysis (GSVA)

Next to the GSEA, the GSVA is an approach to identify the pathway
activities from gene expression data. Differently to the GSEA, it does
not need a reference data frame to compare to. Hence, there was no
expression data provided for comparison in the TCGA analysis, GSVA was
used. There are various solutions to perform GSVA, one of them is
performed by [@GSVA\] by following those five steps. For performing a
GSVA, firstly the cumulative density distribution of a gene over all
samples is estimated. Then the expression statistic of a gene in a
sample based on the cumulative density distribution is calculated to
bring all of the expression values to the same level. The third step is
to rank the genes based on the expression statistic and to normalize the
ranks with z-transformation. The last step is to compute the enrichment
score based on the obtained ranked list. Therefore the
Kolmogorov-Smirnov-like rank statistic is calculated for each gene set.
That is used to calculate the enrichment score for each pathway in each
patient, which is shown a heatmap \[@GSVA].

GSVA is performed with the package "GSVA" [@gsva].

PCA was performed to provide an uncorrelated dataset for the subsequent
UMAP. For the TCGA GSVA pathway activity data the `prcomp()` function
was used. To verify the results, PCA was performed on TCGA expression
data, as well. In this case `Seurat::RunPCA()` from the Seurat package
was used to minimized computation times.

Quelle: seurat package

UMAP analysis was used to identify and visualize clusters in TCGA GSVA
and expression data. This was achieved with the `umap()` function from
the package "umap" \[@umap\] running on all PCs from TCGA GSVA ans
expression data.

### KANN RAUS m.M.n Figure X

To identify pathways with the highest p-Value, obtained from GSVA and
t-testing, a figure x is generated.

For generating figure x, the data from generating a volcano plot is used
to identify the pathways, that are significantly over- or underexpressed
based on the p-value. Pathways with a p-value smaller than 0.025 and a
log2FC bigger than zero are significantly overexpressed, if the log2FC
is smaller than zero, the pathways are significantly underexpressed. In
the next step, the pathways are ranked based on their p-value and the
-log10(p-value) of each pathways is plotted against its rank. One plot
is generated for overexpressed pathways and the other one for under
expressed pathways.

## Regression analysis

### Linear Regression

A linear regression analysis is performed to predict the activity of xxx
based on the activity of the other pathways.

Firstly, the correlation of the pathways for predicting is checked, only
pathways with a low correlation were kept. In the next step, the
variance is checked, 80% of the genes with low variance were omitted.

For the regression analysis only 20% of the pathways were used, to only
use significant pathways.

The regression analysis was tested by

### Neuronal Network

A neural network was used to predict the activity of
REACTOME_INTERLEUKIN_36_PATHWAY based on the activity of other pathways.
Therefore, the network was trained with the pathway activity of 45 xxx
patients from the THCA data for focused analysis. The other 15 patients
were used to validate the network, obtaining a mean squared error (MSE)
value, to evaluate the precision of the network.

For identification of the best initial conditions, 25 different networks
are generated, each one with 2 hidden layers and different combinations
of neurons per layer. For each combination the MSE is calculated and the
3 combinations with the lowest MSE are selected for selection of the
best initial conditions regarding the weights and biases. For each of
the 3 networks 100 random initial conditions are tested, resulting in
one network with the lowest MSE.

## Packages

```{r, eval =TRUE, echo=FALSE, results='hide'}
library(readxl)
packages <- read_excel("~/GitHub/2022-topic-02-team-05/markdown/PackagesRProjekt.xlsx")
```

```{=tex}
\begin{table}[!ht]
    \centering
    \caption{Packages used in the analysis.}
    \begin{tabular}{|m{3cm}|m{3cm}|m{5cm}|m{5cm}|}
    \hline
        Package  & Localisation & Usage & Link \\ \hline
        biomart & pre\_02, pre\_03, pre\_05 & renaming the genenames from the hallmarkpathways-dataframe into ensembleIDs & https://bioconductor.org/packages/release/bioc/html/biomaRt.html \\ \hline
        msigdbr & pre\_03 & downloading all of the canonical pathways and the genes which they include in homo sapiens from the msigbdr data base & https://bioconductor.org/packages/release/data/experiment/html/msigdb.html \\ \hline
        dplyr & pre\_04, pre\_05 & tidying and manipulating of dataframes & https://cran.r-project.org/web/packages/dplyr/index.html \\ \hline
        ggplot2 & pre\_04, pre\_05, descr\_03, descr\_04, THCA\_01, THCA\_02, pan\_02, pan\_04 & allows for the creation of plots with more detailed options & https://cran.r-project.org/web/packages/ggplot2/index.html \\ \hline
        pheatmap & descr\_01, pan\_01, neu\_02, neu\_04 & allows for the creation of heatmaps with more detailed options & https://cran.r-project.org/web/packages/pheatmap/pheatmap.pdf \\ \hline
        vioplot & descr\_02 & creation of violinplots & https://cran.r-project.org/web/packages/vioplot/index.html \\ \hline
        VennDiagram & descr\_05 & creation of VENN-diagrams & https://cran.r-project.org/web/packages/VennDiagram/VennDiagram.pdf \\ \hline
        dplyr & THCA\_01, pan\_01 & ~ & ~ \\ \hline
        fgsea & THCA\_01, pan\_01 & to do a GSEA & https://bioconductor.org/packages/release/bioc/html/fgsea.html \\ \hline
        GSVA & THCA\_01, pan\_03 & to do a GSVA & https://bioconductor.org/packages/release/bioc/html/GSVA.html \\ \hline
        ComplexHeatmap & THCA\_01, pan\_03, pan\_04 & allows for the creation of heatmaps with more detailed options & https://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html \\ \hline
        metaplot & THCA\_02, pan\_02, pan\_04 & data-driven plots & https://cran.r-project.org/web/packages/metaplot/index.html \\ \hline
        gridExtra & THCA\_02, pan\_02, pan\_04 & "implementation of ""grid"" graphics " & https://cran.r-project.org/web/packages/gridExtra/index.html \\ \hline
        umap & THCA\_02, pan\_02, pan\_04 & to do a UMAP & https://cran.r-project.org/web/packages/umap/index.html \\ \hline
        gage & pan\_01 & application of GSEA & https://bioconductor.org/packages/release/bioc/html/gage.html \\ \hline
        psych & pan\_02 & iterative factor analysis & https://cran.r-project.org/web/packages/psych/index.html \\ \hline
        cluster & pan\_04 & cluster analysis & https://cran.r-project.org/web/packages/cluster/cluster.pdf \\ \hline
        MASS & neu\_00 & implementation of neural network & https://cran.r-project.org/web/packages/MASS/index.html \\ \hline
        neuralnet & neu\_03 & training of neural networks & https://cran.r-project.org/web/packages/neuralnet/neuralnet.pdf \\ \hline
        AnnotationDbi & descr\_03 & translating ensemble ids into gennames  & https://bioconductor.org/packages/release/bioc/html/AnnotationDbi.html \\ \hline
        org.Hs.eg.db & descr\_03 & translating ensemble ids into gennames  & https://bioconductor.org/packages/release/data/annotation/html/org.Hs.eg.db.html \\ \hline
    \end{tabular}
\end{table}
```
```{r, eval = TRUE, echo=FALSE}

knitr::kable(packages, format = "markdown", caption= "Packages used in the analysis.")

knitr::kable(packages, format = "latex", caption= "Packages used in the analysis.")

```
