% Options for packages loaded elsewhere
\PassOptionsToPackage{unicode}{hyperref}
\PassOptionsToPackage{hyphens}{url}
%
\documentclass[
]{article}
\author{}
\date{\vspace{-2.5em}}

\usepackage{amsmath,amssymb}
\usepackage{lmodern}
\usepackage{iftex}
\ifPDFTeX
  \usepackage[T1]{fontenc}
  \usepackage[utf8]{inputenc}
  \usepackage{textcomp} % provide euro and other symbols
\else % if luatex or xetex
  \usepackage{unicode-math}
  \defaultfontfeatures{Scale=MatchLowercase}
  \defaultfontfeatures[\rmfamily]{Ligatures=TeX,Scale=1}
\fi
% Use upquote if available, for straight quotes in verbatim environments
\IfFileExists{upquote.sty}{\usepackage{upquote}}{}
\IfFileExists{microtype.sty}{% use microtype if available
  \usepackage[]{microtype}
  \UseMicrotypeSet[protrusion]{basicmath} % disable protrusion for tt fonts
}{}
\makeatletter
\@ifundefined{KOMAClassName}{% if non-KOMA class
  \IfFileExists{parskip.sty}{%
    \usepackage{parskip}
  }{% else
    \setlength{\parindent}{0pt}
    \setlength{\parskip}{6pt plus 2pt minus 1pt}}
}{% if KOMA class
  \KOMAoptions{parskip=half}}
\makeatother
\usepackage{xcolor}
\IfFileExists{xurl.sty}{\usepackage{xurl}}{} % add URL line breaks if available
\IfFileExists{bookmark.sty}{\usepackage{bookmark}}{\usepackage{hyperref}}
\hypersetup{
  hidelinks,
  pdfcreator={LaTeX via pandoc}}
\urlstyle{same} % disable monospaced font for URLs
\usepackage[margin=1in]{geometry}
\usepackage{graphicx}
\makeatletter
\def\maxwidth{\ifdim\Gin@nat@width>\linewidth\linewidth\else\Gin@nat@width\fi}
\def\maxheight{\ifdim\Gin@nat@height>\textheight\textheight\else\Gin@nat@height\fi}
\makeatother
% Scale images if necessary, so that they will not overflow the page
% margins by default, and it is still possible to overwrite the defaults
% using explicit options in \includegraphics[width, height, ...]{}
\setkeys{Gin}{width=\maxwidth,height=\maxheight,keepaspectratio}
% Set default figure placement to htbp
\makeatletter
\def\fps@figure{htbp}
\makeatother
\setlength{\emergencystretch}{3em} % prevent overfull lines
\providecommand{\tightlist}{%
  \setlength{\itemsep}{0pt}\setlength{\parskip}{0pt}}
\setcounter{secnumdepth}{-\maxdimen} % remove section numbering
\ifLuaTeX
  \usepackage{selnolig}  % disable illegal ligatures
\fi

\begin{document}

\hypertarget{material-and-methods}{%
\section{Material and Methods}\label{material-and-methods}}

In the following, two analyses are performed: a pan cancer analysis and
a focused analysis about THCA.

\hypertarget{our-data-sets}{%
\subsection{Our data sets}\label{our-data-sets}}

For the analysis four data sets were provided.

The first data set is a Gene expression data frame. The Gene expression
data frame contains 60,000 genes and and their expression in 10,000
patients. It is derived from The Cancer Genome Atlas (TCGA). The
expression of the genes was obtained by RNA-seq.

The second data frame contains 37 clinical annotations like Tumor type,
age, gender, etc. for each of the 10,000 patients from the Gene
expression data frame.

The third object is a list that contains five lists for the focused
analysis, one list for each tumor type (BRCA, KIRC, LUAD, PRAD, THCA).
For the focused analysis, the THCA data (only DTC) was used. The THCA
data contains 3 data frames, each one with information about the same 60
patients. The first data is a gene expression matrix from THCA tissue,
the second data contains the gene expression from normal tissue and the
third data frame contains the clinical annotations like age and gender.
Gene expression data was obtained by RNA-seq.

The fourth object contains 46 pathways involved in phenotypes partly
included in the hallmarks of cancer and the genes involved in those
pathways.

SIND DIE DATEN NORMALISIERT --\textgreater{} Normalisiert glaub ich
(Anna) ODER ALS COUNTS?

\hypertarget{metabolic-pathway-selection}{%
\subsection{Metabolic pathway
selection}\label{metabolic-pathway-selection}}

From the Molecular Signature Database (MSigDB) \cite{xxx} metabolic
pathways were selected. First, they were compared to the given
Hallmark-Pathways in order to select pathways that differ to the
Hallmark-Pathways. The goal was to identify more pathways, that are
important for the development of cancer. Therefore it was important that
as many genes from the selected pathways as possible are also included
in the provided Hallmark pathways. To identify the relevant pathways,
the intersection of genes was calculated and the genes with an
intersection of at least 99\% were maintained for further analysis.

xxx??????????????????????

To avoid duplicates in between the metabolic pathways and between the
Hallmark pathways and the metabolic pathways, the pathways were checked
for duplicates with the Jaccard index. Pathways with a sum of Jaccard
indices beyond the 1-sigma range were discarded.

\hypertarget{preprocessing}{%
\subsection{Preprocessing}\label{preprocessing}}

\hypertarget{deleting-not-available-values-nas}{%
\subsubsection{Deleting Not Available Values
(NA's)}\label{deleting-not-available-values-nas}}

Deleting of NA's was done with the R-function na.omit(x).

\hypertarget{low-variance-filtering}{%
\subsubsection{Low-Variance Filtering}\label{low-variance-filtering}}

Low variance filtering is performed to delete genes with a low variance
in gene expression from the data set. It is performed to delete genes
that are expressed the same in all cancer types (pancancer analysis) or
the same in normal cells. To calculate the variance of the gene
expression of a gene, the r-function var(x) is used and genes with a
lower variance than a certain threshold value are
removed.\textbackslash{} For the focused analysis the variance of the
gene expression for each gene in tumor tissue was calculated. Genes with
a variance beneath a certain threshold were deleted in the data sets of
tumor and normal tissue.

\hypertarget{biotype-filtering}{%
\subsubsection{Biotype Filtering}\label{biotype-filtering}}

The biotype filtering was conducted for the pancancer data and the
focussed analysis data. The biotype of each gene was determined (protein
coding, RNA,\ldots) and compared with the biotypes of pathways. To allow
an appropriate comparison of the expression data and further reduce the
data, only biotypes were kept that are available in the pathways. The
biotype can be determined with the R-function checkbiotypes(x) from the
package biomaRt {[}@biomaRt{]}.

\hypertarget{selection-of-metabolic-pathways}{%
\paragraph{Selection of metabolic
pathways}\label{selection-of-metabolic-pathways}}

(da eine hohe jaccard summe eine hohe überschneidung mit anderen
pathways bedeutet. In einer heatmap sind hohe Jacccard indices weiß bis
rot gefährbt. Ein niedriger Jaccard index ist blau gefärbt.)

To test for duplicate pathways in the selected metabolic pathways
compared to the hallmark pathways and the compared tp the metabolic
pathways themselves, the Jaccard index between to pathways were
calculated.\textbackslash{} There were a few duplicates between the
metabolic and Hallmark pathways. Those metabolic pathways with a high
Jaccard index were discarded. The success of the cleaning was checked by
again calculating the Jaccard index between the metabolic and the
hallmark pathways. The values of the Jaccard index were then illustrated
in a heatmap \ref{xxxfigure heatmap}. It can be assumed, that the
selection of relevant pathways was successful because the pathways
differ between each other. The number of metabolic pathways could be
reduced from xxx to 600.

\hypertarget{descriptive-analysis}{%
\subsection{Descriptive analysis}\label{descriptive-analysis}}

\hypertarget{mean-variance-plot}{%
\subsubsection{Mean-variance plot}\label{mean-variance-plot}}

In a mean-variance plot the variance is plotted over the mean of
expression values of the single genes across all patients. Thus, the
variance and mean were calculated by the R-functions var(x) and mean(x).
This is done to determine genes, which differ a lot in their expression
levels across all patients. The plot is created with the package
\ref{ggplot2} xxx.

\hypertarget{violin-plot}{%
\subsubsection{Violin plot}\label{violin-plot}}

To check the distribution of a data set and compare it with other data
sets violin plots are used. Based on how similar the violin plots are,
it can be implied that the data is normalized. Violin plots are tilted
and mirrored density plots of gene expression values. The y-axis shows
the gene expression value and the x-axis shows the amount of genes with
a certain gene expression value.

\hypertarget{jaccard-index}{%
\subsubsection{Jaccard-Index}\label{jaccard-index}}

The Jaccard-Index is a method to describe the similarity between two
quantities. It is computed via dividing the union by the intersection.
This is used to determine the degree in which metabolic pathways are
similar to each other.

\hypertarget{volcano-plot}{%
\subsubsection{Volcano plot}\label{volcano-plot}}

A volcano plot is used to identify significantly differentially
expressed genes. This is done to determine genes or pathways, which are
up- or down- regulated in tumor tissue vs.~normal tissue. The mean of
each gene is calculated for normal and THCA tissue and used for the
calculation of the Log2-Foldchange (Log2FC) in the following way, since
the provided expression data is already log2 data:

\[
log2FC = mean(normal tissue) - mean(tumor tissue)
\] In the next step, a two-sided t-test was performed to determine the
significance of a difference in expression.

To avoid the accumulation of type 1 errors, a bonferroni correction was
performed. n is the number of genes in the cleaned data set for focused
analysis:

\[
\alpha = \frac{0.025}{n}
\]

In the volcano plot the -log10 of the calculated p values is plotted
against die Log2FC. Genes with a a lower p-value than the corrected
alpha-value are significantly differently expressed. If the Log2FC is
additionally higher than 0.1, the genes are significantly over expressed
in tumor tissue, if the Log2FC is higher lower than -0.1, the genes are
significantly under expressed in tumor tissue.

\hypertarget{data-reduction-and-pathway-activities}{%
\subsection{Data Reduction and Pathway
Activities}\label{data-reduction-and-pathway-activities}}

\hypertarget{pca}{%
\subsubsection{PCA}\label{pca}}

The package xxx is used to perform the PCA. Therefore the data obtained
from the GSEA was used. After performing the PCA, the results were
plotted to visualize the different clusters.

The PCA was performed for pathway and gene activity. For analysis of the
gen activity the package xxx was used. Dazu wurde noch analysiert, wie
die Pathways auf die PCs verteilt sind.

\hypertarget{umap}{%
\subsubsection{UMAP}\label{umap}}

Like PCA, UMAP is a technique to reduce dimensions and to understand and
visualize high dimensional data sets. Compared to PCA, UMAP better
preserves the global structure and is much faster than other comparable
techniques (for example t-SNE \cite{xxx}). The algorithm starts by
setting up a high-dimensional graph representation of the data. From
each data point, a radius is extended and when two radii come into
contact the points are connected. The radius is chosen individually for
each point based on the distance to the nearest neighbor. The algorithm
does not stop before every point is not connected at least to its
closest neighbor. The resulting clustered high-dimensional graph is then
optimized for a visualization in low-dimensions. Using this technique,
the pan-cancer data is visualized.

\hypertarget{gsea}{%
\subsubsection{GSEA}\label{gsea}}

The GSEA is used to identify enriched pathways in tumor tissue. Next to
the tumor tissue data, the THCA data includes also a normal tissue gene
expression data frame which is used as a reference for activity
comparison.

First, the log2FC is calculated for every gene of each each patient and
is then ranked in a vector. This vector begins with the highest log2FC
and ends with the lowest. A high log2FC implies that the this gene is
higher expressed in tumor tissue compared to normal tissue in this
particular patient.

Using the ranked log2FC vectors, the activity of each pathway for the
patient is calculated. By iterating over every gene of the ranked
vector, it was checked if it lies or does not lie in a particular
pathway. If a gene lies in the pathway, the log2FC value is summed up to
a running sum. If the gene does not lie in the pathway, the log2FC value
is subtracted from the running sum. Therefore, when a pathway is highly
expressed compared to normal tissue, the the running sum scores a high
value in the beginning and decreases to the end of the iteration. This
results in a cumulative function that has a peak at a certain place. At
this index of the ranked vector, the expression value of the
corresponding gene is taken as the enrichment score of the analysed
pathway and the patient belonging to the used vector. This process is
then repeated for each pathway and each patient.

\hypertarget{gsva}{%
\subsubsection{GSVA}\label{gsva}}

Next to the GSEA, the GSVA is an approach to identify the pathway
activities from gene expression data. Differently to the GSEA, it does
not need a reference data frame to compare to.\textbackslash{} Hence,
there was no expression data provided for comparison in the TCGA
analysis, GSVA was used. There are various solutions to perform GSVA,
one of them is performed by Hänzelmann et al xxx by following those five
steps.\textbackslash{} For performing a GSVA, firstly the cumulative
density distribution of a gene over all samples is estimated. Then the
expression statistic of a gene in a sample based on the cumulative
density distribution is calculated to bring all of the expression values
to the same level. The third step is to rank the genes based on the
expression statistic and to normalize the ranks with z-transformation.
The last step is to compute the enrichment score based on the obtained
ranked list. Therefore the Kolmogorov-Smirnov-like rank statistic is
calculated for each gene set. That is used to calculate the enrichment
score for each pathway in each patient, which is shown a heatmap.
(Hänzelmann, Castelo, and Guinney 2013) xxx Hänzelmann, Sonja, Robert
Castelo, and Justin Guinney. 2013. ``GSVA: Gene Set Variation Analysis
for Microarray and Rna-Seq Data.'' Journal Article. BMC Bioinformatics
14 (1): 7. \url{https://doi.org/10.1186/1471-2105-14-7.}

\hypertarget{figure-x}{%
\subsubsection{Figure X}\label{figure-x}}

To identify pathways with the highest p-Value, obtained from GSVA and
t-testing, a figure x is generated.

For generating figure x, the data from generating a volcano plot is used
to identify the pathways, that are significantly over- or underexpressed
based on the p-value. Pathways with a p-value smaller than 0.025 and a
log2FC bigger than zero are significantly overexpressed, if the log2FC
is smaller than zero, the pathways are significantly underexpressed. In
the next step, the pathways are ranked based on their p-value and the
-log10(p-value) of each pathways is plotted against its rank. One plot
is generated for overexpressed pathways and the other one for under
expressed pathways.

\hypertarget{linear-regression}{%
\subsubsection{Linear Regression}\label{linear-regression}}

\hypertarget{neuronal-network}{%
\subsubsection{Neuronal Network}\label{neuronal-network}}

A neural network was used to predict the activity of
REACTOME\_INTERLEUKIN\_36\_PATHWAY based on the activity of other
pathways. Therefore, the network was trained with the pathway activity
of 45 xxx patients from the THCA data for focused analysis. The other 15
patients were used to validate the network, obtaining a mean squared
error (MSE) value, to evaluate the precision of the network.

For identification of the best initial conditions, 25 different networks
are generated, each one with 2 hidden layers and different combinations
of neurons per layer. For each combination the MSE is calculated and the
3 combinations with the lowest MSE are selected for selection of the
best initial conditions regarding the weights and biases. For each of
the 3 networks 100 random initial conditions are tested, resulting in
one network with the lowest MSE.

\hypertarget{packages}{%
\subsection{Packages}\label{packages}}

\begin{verbatim}
## Warning: Paket 'readxl' wurde unter R Version 4.1.3 erstellt
\end{verbatim}

==============
====================================================================
=====================================================================================================================
================================================================================
Package Lokalisation Verwendung Link\\
==============
====================================================================
=====================================================================================================================
================================================================================
biomart pre\_02, pre\_03, pre\_05 renaming the genenames from the
hallmarkpathways-dataframe into ensembleIDs
\url{https://bioconductor.org/packages/release/bioc/html/biomaRt.html}\\
msigdbr pre\_03 downloading all of the canonical pathways and the genes
which they include in homo sapiens from the msigbdr data base
\url{https://bioconductor.org/packages/release/data/experiment/html/msigdb.html}\\
dplyr pre\_04, pre\_05 tidying and manipulating of dataframes
\url{https://cran.r-project.org/web/packages/dplyr/index.html}\\
ggplot2 pre\_04, pre\_05, descr\_03, descr\_04, THCA\_01, THCA\_02,
pan\_02, pan\_04 allows for the creation of plots with more detailed
options
\url{https://cran.r-project.org/web/packages/ggplot2/index.html}\\
pheatmap descr\_01, pan\_01, neu\_02, neu\_04 allows for the creation of
heatmaps with more detailed options
\url{https://cran.r-project.org/web/packages/pheatmap/pheatmap.pdf}\\
vioplot descr\_02 creation of violinplots
\url{https://cran.r-project.org/web/packages/vioplot/index.html}\\
VennDiagram descr\_05 creation of VENN-diagrams
\url{https://cran.r-project.org/web/packages/VennDiagram/VennDiagram.pdf}\\
dplyr THCA\_01, pan\_01 NA NA\\
fgsea THCA\_01, pan\_01 to do a GSEA
\url{https://bioconductor.org/packages/release/bioc/html/fgsea.html}\\
GSVA THCA\_01, pan\_03 to do a GSVA
\url{https://bioconductor.org/packages/release/bioc/html/GSVA.html}\\
ComplexHeatmap THCA\_01, pan\_03, pan\_04 allows for the creation of
heatmaps with more detailed options
\url{https://bioconductor.org/packages/release/bioc/html/ComplexHeatmap.html}\\
metaplot THCA\_02, pan\_02, pan\_04 data-driven plots
\url{https://cran.r-project.org/web/packages/metaplot/index.html}\\
gridExtra THCA\_02, pan\_02, pan\_04 implementation of ``grid'' graphics
\url{https://cran.r-project.org/web/packages/gridExtra/index.html}\\
umap THCA\_02, pan\_02, pan\_04 to do a UMAP
\url{https://cran.r-project.org/web/packages/umap/index.html}\\
gage pan\_01 application of GSEA
\url{https://bioconductor.org/packages/release/bioc/html/gage.html}\\
psych pan\_02 iterative factor analysis
\url{https://cran.r-project.org/web/packages/psych/index.html}\\
cluster pan\_04 cluster analysis
\url{https://cran.r-project.org/web/packages/cluster/cluster.pdf}\\
MASS neu\_00 implementation of neural network
\url{https://cran.r-project.org/web/packages/MASS/index.html}\\
neuralnet neu\_03 training of neural networks
\url{https://cran.r-project.org/web/packages/neuralnet/neuralnet.pdf}\\
AnnotationDbi descr\_03 translating ensemble ids into gennames
\url{https://bioconductor.org/packages/release/bioc/html/AnnotationDbi.html}\\
org.Hs.eg.db descr\_03 translating ensemble ids into gennames
\url{https://bioconductor.org/packages/release/data/annotation/html/org.Hs.eg.db.html}
==============
====================================================================
=====================================================================================================================
================================================================================

\end{document}
